recipe-server/client/control/tests/components/test_RecipeForm.js

```diff
@@ -33,12 +33,14 @@ describe('<RecipeForm>', () => {
       expect(wrapper.find(ConsoleLogFields).length).toBe(1);
     });
 
-    it('should render the fields for the show-heartbeat action', () => {
-      const wrapper = shallow(
-        <RecipeForm selectedAction="show-heartbeat" {...propFactory()} />
-      );
-      expect(wrapper.find(HeartbeatFields).length).toBe(1);
-    });
+    // TODO(osmose): Figure out how to shallow-render and test connected components
```

Comment:
If I use mount, I get an error:

```
Error: Field must be inside a component decorated with reduxForm() in client/control/tests/index.js line 140 > eval (line 59)
```

The shallow render of `RecipeForm` elsewhere works because it's not a connected component (vs the default export, which I don't import during these tests), and the `Field` components never actually get rendered.

I suspect the long-term solution is either use mount and get better at mocking the state tree so that we can fully render connected components during tests, or do some wrapper around the heartbeat fields with a classname we can hook into. The former is definitely the more desirable option, but also conflicts with the upcoming state tree changes.